lib/src/excel.dart

```diff
@@ -101,9 +144,15 @@ class ExcelDart {
 
       // Conditionally format the budget column once the table is set.
       _formatBudgetColumn(tableName);
+
+      // Highlight underpacing insertion orders if highlightUnderpacing is true.
+      if (highlightUnderpacing) _highlightUnderpacingRows(tableName);
     }));
   }
 
+  static String _getExcelColumnReference(int offset) =>
+      String.fromCharCode(_tableColumnStartAddress.codeUnitAt(0) + offset);
+
   /// Conditionally formats the Budget column based on Budget Unit.
   ///
   /// 'RC[-2]' in the formula follows Excel's Relative Notation and references
```

Comment:
please create issue to refactor this from hardcoded relative to calculated absolute. (also elsewhere where applicable, and drop issue here.)

Reply:
sorry totally forgot to address this :(